src/test/java/com/squareup/javapoet/UtilTest.java

```diff
@@ -15,61 +15,87 @@
  */
 package com.squareup.javapoet;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.DynamicContainer.dynamicContainer;
+import static org.junit.jupiter.api.DynamicTest.dynamicTest;
 
-import org.junit.Test;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DynamicNode;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
```

Comment:
Going from 2 imports to 9 makes me want to back away from Jupiter forever.

Reply:
+7 more imports to go from

![image](https://user-images.githubusercontent.com/2319838/45936483-4e659280-bfb7-11e8-9532-5d704e8ce7b7.png)

to

![image](https://user-images.githubusercontent.com/2319838/45936464-2e35d380-bfb7-11e8-9e5d-2656d8e41aa0.png)

Better reporting needs some imports.

But as I stated in the original description text, I'll rework this commit to only show minimal changes.